src/styles/blocks/card.scss

```diff
@@ -0,0 +1,79 @@
+.card {
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  width: 200px;
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__photo {
+    display: block;
+    margin: 0 auto 40px;
+
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-name {
+    font-size: 12px;
+    font-weight: 500;
+    color: $text-color-one;
+    padding-bottom: 4px;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+    color: $text-color-two;
+  }
+
+  &__product-price {
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+
+    &-name {
+      color: $text-color-two;
+      font-weight: 400;
+      font-size: 12px;
+    }
+
+    &-cost {
+      color: $text-color-one;
+      padding-bottom: 16px;
+      font-weight: 700;
+      font-size: 16px;
+    }
+  }
+
+  &__product-buy {
+    a {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+
+      font-size: 14px;
+      font-weight: 700;
+      color: #fff;
+      background-color: #00acdc;
+
+      box-sizing: border-box;
+      width: 166px;
```

Comment:
Don't fix container size (if there is no such a requirement)
<img width="570" alt="image" src="https://github.com/user-attachments/assets/2dca66d4-be1c-4deb-9d76-8d4d5e8944a2">